GliaWidgets/Sources/CallVisualizer/Coordinator/CallVisualizer.Coordinator.swift

```diff
@@ -15,12 +16,19 @@ extension CallVisualizer {
             bubbleView.pan = { [weak self] translation in
                 self?.updateBubblePosition(translation: translation)
             }
+
+            interactorSubscription = environment.interactorPublisher
```

Comment:
Could you add a test checking that `CallVisualizer.Coordinator.activeInteractor` is updated once Glia.sharedInstance.interactor` is re-created?

Reply:
fixed [00a5146](https://github.com/salemove/ios-sdk-widgets/pull/1198/commits/00a51464b7b980c0480ffd18093c4f2cf2a682f6)

Reply:
Awesome! Thanks!